blocks/article-list/article-list.js

```diff
@@ -83,8 +82,7 @@ export default async function listArticles(block, config = { filter: null, maxEn
     contextFilter = determineContextFilter();
   }
 
-  let articles = await ffetch(ARTICLE_INDEX)
-    .filter(contextFilter);
+  let articles = await ffetch(ARTICLE_INDEX).filter(contextFilter);
```

Comment:
I know the variable `ARTICLE_INDEX` was there before but it is useless as only used once should we remove it?